Cargo.toml

```diff
@@ -38,7 +38,8 @@ dusk-core = "0.1.0"
 dusk-vm = "0.1.0"
 # dusk-vm = { version = "0.1.0", path = "./vm/" }
 node = { version = "0.1.0", path = "./node/" }
-node-data = { version = "0.1.0", path = "./node-data/" }
+node-data = { version = "1.0.0", package = "dusk-node-data" }
+# node-data = { version = "1.0.0", path = "./node-data/", package = "dusk-node-data" }
```

Comment:
Left over?

Reply:
I left there on purpose to be switched to dev dep if needed